parent/pom.xml

```diff
@@ -337,6 +337,24 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.jboss.tools.tycho-plugins</groupId>
+				<artifactId>repository-utils</artifactId>
+				<version>${jbossTychoPluginsVersion}</version>
+				<executions>
+					<execution>
+						<goals>
+							<goal>compare-version-with-baselines</goal>
+						</goals>
+						<phase>verify</phase>
+						<configuration>
+							<baselines>
+								<repo>http://download.jboss.org/jbosstools/updates/stable/luna</repo>
```

Comment:
Please extract this into a property so it can be more easily changed after each release.